pages/docs/cloud/repository-integration.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Grouparoo Cloud Repository Integration"
+date: "2021-11-10"
+pullQuote: "Simple integration steps for Grouparoo Cloud"
+---
+
+If you are here you should already have an API Key and Project ID.
+
+Grouparoo Cloud requires three things to function: A github action yaml file and two secrets to identify and authorize the code config repository.
+
+### Github Repository Secrets
+
+Grouparoo Cloud requires two secrets to be available in your repository: `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID`.
+
+If these values have already been provided for you, skip ahead. Otherwise you can get them from [Grouparoo Cloud](https://cloud.grouparoo.com/) in your Project's Integrations page.
+
+In your Github repository go to: Settings -> Secrets -> New repository secret.
+
+Add `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID` with the provided values.
+
+<sub>The Project ID should look similar to `prj_some-set-of-values`.</sub>
+
+### Github Action
+
+We recommend creating a new branch for this since our system can check PR builds for validity, saving you some potential headache.
```

Comment:
The addition of the github action doesn't necessarily need to be on a new branch- at this point in the process there's nothing to validate yet. I think this can be an opportunity to explain a bit about what this action does and why you may want to install it, highlighting that Grouparoo Cloud can auto validate the config changes through PRs as well as deploy them once merged into main.

Reply:
The thought for these docs is you already have an existing code config repo. I've tried to clarify that at the beginning.